packages/fast-components-react-base/src/context-menu-item/context-menu-item.spec.tsx

```diff
@@ -19,6 +19,11 @@ describe("context menu item", (): void => {
         expect((ContextMenuItem as any).name).toBe(ContextMenuItem.displayName);
     });
 
+    test("should have root element type div", () => {

```

Comment:
I don't actually think we need this test, it's not an accessibility check like the ones we have for "role" and "aria-label".